app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -135,24 +135,6 @@ def document_class_prefix
     'blacklight-'
   end
 
-  ##
-  # Render the sidebar partial for a document
-  #
-  # @param [SolrDocument] document
-  # @return [String]
-  def render_document_sidebar_partial(document)
```

Comment:
geoblacklight is using this helper to inject additional sidebar content:

https://github.com/geoblacklight/geoblacklight/blob/7d3c31c7af3362879b97e2c1351a2496c728c59c/app/helpers/blacklight_helper.rb#L7